src/yunohost/hook.py

```diff
@@ -294,6 +295,13 @@ def hook_callback(action, hooks=[], args=None, no_trace=False, chdir=None,
                 result[state][name].append(path)
             except KeyError:
                 result[state][name] = [path]
+
+            #print(hook_return)
+            #for r in hook_return.:
+            result['stdreturn'].extend(hook_return) #for r in hook_return
```

Comment:
I would rather have `result[name]['stdreturn']` (in anticipation of the new diagnosis system ;P)
And I would propagate this hierarchy also for the state a few line before to have `result[name][state]`  ... which in turns need to be propagated where this `result` value is used (I found backup and service)